packages/git/src/browser/git-view-contribution.ts

```diff
@@ -160,8 +160,10 @@ export class GitViewContribution extends AbstractViewContribution<GitWidget> {
             isEnabled: () => !!this.repositoryTracker.selectedRepository
         });
         registry.registerCommand(GIT_COMMANDS.CHANGE_REPOSITORY, {
-            execute: () => this.quickOpenService.changeRepository(),
-            isEnabled: () => this.repositoryTracker.allRepositories.length > 1
+            execute: () => {
+                this.quickOpenService.changeRepository();
+            },
```

Comment:
FYI: you could write `execute: () => this.quickOpenService.changeRepository(),` too.

Reply:
Sure will do